dotcom-rendering/src/lib/liveblogAdSlots.ts

```diff
@@ -20,10 +22,9 @@ const MAX_INLINE_ADS = 8;
 const MIN_SPACE_BETWEEN_ADS = 1_500;
```

Comment:
We had a positive result on desktop with the value in the previous AB test. Ad-ratio up 6%: https://docs.google.com/spreadsheets/d/16DLzkrj4BgyU3iVus2Vd_90z8XGnvsMx1vpljqlZ0H0/edit#gid=745321536